docs/developer.md

```diff
@@ -221,9 +350,26 @@ npm run dev
 
 Note that you can run `npm run dev` from within any folder if you'd like to watch individual packages instead of the entire directory structure.
 
-### Practices we follow
+(myst-api-server)=
+## The MyST API server
+
+We run a lightweight server at [`api.mystmd.org`](https://api.mystmd.org/).
+This is primarily used for resolving template paths and allowing builds to download them.
+
+For example, to get a list of template types you can `GET` this URL:
+
+https://api.mystmd.org/templates
+
+And to see a list of available templates that can be resolved for Typst, you can `GET` this URL:
+
+https://api.mystmd.org/templates/typst
+
+:::{error} todo - Document how the API server is configured and where it is run.
```

Comment:
It is here:
https://github.com/myst-templates/templates

Again, no config.

We have also discussed removing this, which would enable the CLI to be more able to run off-line as well as removing a complexity/dependency.

Reply:
ah this is super helpful, thanks for pointing that out. I will add it!

I know that it seems trivial, but to folks that don't have the knowledge of "where does this thing exist? who maintains it? how is it configured?", explicitly documenting "there is no configuration for this and the logic is over here" is extremely helpful to others.